test/co19/line_based_doc_comments_test.dart

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A line-based doc comment is a comment that starts with `///`. One
+/// or more consecutive lines that begin with `///` form a doc comment block.
+///
+/// The block continues even if interrupted by single-line non-doc comments
+/// (lines starting with `//`) or by blank lines. The interrupting lines are
+/// ignored when extracting documentation text.
+///
+/// For each line that begins with `///`, the parser removes the three slashes
+/// and all leading whitespace to produce the documentation text. Exception:
+/// inside fenced code blocks (` ``` `), whitespace after the leading `///` is
+/// preserved to maintain code formatting.
+/// @author sgrekhov22@gmail.com
+
```

Comment:
Would it make sense to consider this comment as a DartDoc comment on the library as such?
```suggestion
library;

```

Reply:
Thanks! makes sense. Committed the suggestion.